test/integration/targets/test_infra/runme.sh

```diff
@@ -22,9 +22,14 @@ echo "ensure playbook output shows assert/fail works (True)"
 echo "$PB_OUT" | grep -F "fail works (True)" || exit 1
 echo "$PB_OUT" | grep -F "assert works (True)" || exit 1
 
+set -e
+
 # ensure test-module script works well
 PING_MODULE_PATH="$(pwd)/../../../../lib/ansible/modules/system/ping.py"
 ../../../../hacking/test-module -m "$PING_MODULE_PATH" -I ansible_python_interpreter="$(which python)"
 
+# ensure module.ansible_version is defined when using test-module
+../../../../hacking/test-module -m library/test.py <<< '{"ANSIBLE_MODULE_ARGS": {}}'
```

Comment:
-I ansible_python_interpreter="$(which python)"   is missing, likely why the test fail on Freebsd.